package.json

```diff
@@ -33,7 +33,7 @@
     "db:seed:reset": "(cd utils && npm run db:seed:reset)",
     "env:copy": "(cd utils && npm run env:copy)",
     "housekeeping": "npm run update:all && npm run install:all && npm run prettier:fix:all && npm run lint:fix:all && npm run validate:yml:all && npm run audit:fix:all && npm run spellcheck",
-    "install:all": "npm install --if-present",
+    "install:all": "npm install --workspaces --if-present",
```

Comment:
Not sure this is needed - https://github.com/UK-Export-Finance/dtfs2/blob/main/doc/npm-workspaces.md#notes-on-workspaces-flags